1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/module/modules/world/Scaffold.java

```diff
@@ -322,10 +322,17 @@ private void place() {
         if (!delayTimer.hasTimePassed(delay) || (sameYValue.get() && launchY - 1 != (int) targetPlace.getVec3().yCoord))
             return;
 
-        int blockSlot = -1;
+        int blockSlot = mc.thePlayer.inventory.currentItem + 36;
         ItemStack itemStack = mc.thePlayer.getHeldItem();
 
-        if (mc.thePlayer.getHeldItem() == null || !(mc.thePlayer.getHeldItem().getItem() instanceof ItemBlock)) {
+        //Fixed 0 Item Bug
+        //If you are too fast you will fall down, because of the bug
+        if (mc.thePlayer.getHeldItem() == null || !(mc.thePlayer.getHeldItem().getItem() instanceof ItemBlock) || mc.thePlayer.getHeldItem().stackSize <= 0) {
+
+            if (mc.thePlayer.getHeldItem() != null && mc.thePlayer.getHeldItem().stackSize <= 0) {
+                mc.thePlayer.inventory.removeStackFromSlot(mc.thePlayer.inventory.currentItem);
+            }
```

Comment:
Can you extract this to a method and use it instead of this if statement? This would make it easier to read. Like `public static void removeIfStackEmpty(ItemStack)`